File: src/Front/app/src/main/java/com/example/front/model/DTO/WorkingHoursDTO.kt
package com.example.front.model.DTO

import com.nimbusds.jose.shaded.gson.annotations.SerializedName

data class WorkingHoursDTO(
    @SerializedName("shopId") val shopId: Int,
    @SerializedName("day") val day: Int,
    @SerializedName("openingHours") val openingHours: String,
    @SerializedName("closingHours") val closingHours: String,
    @SerializedName("shop") val shop: String?
)

File: src/back/BLL/Dtos/HelpModels/WorkingHoursDto.cs
namespace back.BLL.Dtos.HelpModels
{
    public class WorkingHoursDto
    {
        public DayOfWeek Day { get; set; }
        public string OpeningHours { get; set; }
        public string ClosingHours { get; set; }
    }
}
